src/training/day5/Student.java

```diff
@@ -0,0 +1,74 @@
+package training.day5;
+
+import java.util.*;
+
+
+class Student extends Person {
+    private String classStudent;
+    private int numberInClass;
+    private List<String> subjects;
+    private List<Double> grades;
```

Comment:
Add new line after the field. Separate with new line fields, constructors and methods.